notebooks/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -31,15 +31,23 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # two hidden layers of [32, 8] just in like the BQML DNN
</pre><p>remove reference to BQML DNN here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good catch! Thanks.</p>